x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_lifecycle/helpers/format_bytes.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const units = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'] as const;
+type SizeUnitTuple = typeof units;
+export type SizeUnit = SizeUnitTuple[number];
+
+export const formatBytes = (bytes: number, unit?: SizeUnit) => {
```

Comment:
Instead of creating a dedicated formatter, you could use `formatNumber` from `@elastic/eui`